docs/user/howto/pull.md

```diff
@@ -0,0 +1,56 @@
+# Pulling charts to the local machine
+
+Sure, installing charts from an url or repo is cool, but sometimes you need to modify those a bit locally or just have a look at what the chart is actually doing.
+
+For that you can use `hypper pull` and that will download the chart to your local machine
+
+
+## Pulling a chart
+
+```shell
+$ hypper pull hypper/fleet
+$ ls fleet*
+fleet-0.3.500.tgz
+```
+
+## Pulling a chart and extracting it
+
+Usually pull will download the tar.gz chart package, but you can use `--untar` so instead the package is extracted automatically
```

Comment:
```suggestion
Usually pull will download the tar.gz chart package, but you can use `--untar` so instead the package is extracted automatically.
```
I would either add dots (or maybe colons) to all the phrases, or remove the dot in "Pulling a specific chart version".